machine_learning/management/commands/ml_smoothing.py

```diff
@@ -405,7 +405,7 @@ def initial_processing(
                 "reference_cycle": all_data[Key.REF_ALL_MATS][Key.N],
                 Key.COUNT_MATRIX:
                     all_data[Key.REF_ALL_MATS][Key.COUNT_MATRIX],
-                "cycle": main_data[Key.N],
+                Key.CYC: main_data[Key.N],
```

Comment:
now, I am slightly confused.

I thought you had replaced Key.N by Key.CYC
now I discover that they are different things...
what is the difference?

Reply:
Not sure. They were different Strings, so I gave them different Keys. They might be the same thing. Right now `N = "cycle_number"` and `CYC = "cycle"`

Reply:
I added a TODO